packages/inter-protocol/src/runStake/runStakeKit.js

```diff
@@ -319,7 +321,7 @@ const helperBehavior = {
   },
 
   /**
-   * Given sufficient RUN payoff, refund the attestation.
+   * Given sufficient Minted payoff, refund the attestation.
```

Comment:
Everywhere else in this file it's **Debt**. Why does this case turn into **Minted**?

Reply:
IIRC just an incomplete move from Debt terminology.